From 1e5d499703c535565c9b49dceba86ee4e6118174 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Wed, 28 Nov 2018 15:10:32 +0200
Subject: [PATCH 5231/5242] Revert "MA-12957: arm64: dts: imx8qm/qxp mek:
 Configure interrupts for adv7535"

commit  d5968fe8d2b91e67ec50b3bebbb950622e3ea046 from
https://source.codeaurora.org/external/imx/linux-imx.git

This commit is breaking suspend/resume, so reverting it until a fix is
provided.

This reverts commit 47427ff4ce37a6d6e0468de9b6e61f8a58b21d9a.
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi  |    6 ------
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi |   14 ++++----------
 2 files changed, 4 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
index ccb9460..afb8032 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
@@ -593,7 +593,6 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc600004c
 				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA	0xc600004c
-				SC_P_MIPI_DSI0_GPIO0_00_MIPI_DSI0_GPIO0_IO00	0x00000020
 			>;
 		};
 
@@ -601,7 +600,6 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL	0xc600004c
 				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA	0xc600004c
-				SC_P_MIPI_DSI1_GPIO0_00_MIPI_DSI1_GPIO0_IO00	0x00000020
 			>;
 		};
 
@@ -1363,8 +1361,6 @@
 		reg = <0x3d>;
 		adi,dsi-lanes = <4>;
 		adi,dsi-channel = <1>;
-		interrupt-parent = <&gpio0>;
-		interrupts = <1 IRQ_TYPE_LEVEL_LOW>;
 		status = "okay";
 
 		port {
@@ -1407,8 +1403,6 @@
 		reg = <0x3d>;
 		adi,dsi-lanes = <4>;
 		adi,dsi-channel = <1>;
-		interrupt-parent = <&gpio0>;
-		interrupts = <1 IRQ_TYPE_LEVEL_LOW>;
 		status = "okay";
 
 		port {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
index 772bd71..57164b5 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
@@ -454,9 +454,8 @@
 
 		pinctrl_i2c0_mipi_lvds0: mipi_lvds0_i2c0_grp {
 			fsl,pins = <
-				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc6000020
-				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA	0xc6000020
-				SC_P_MIPI_DSI0_GPIO0_00_MIPI_DSI0_GPIO0_IO00	0x00000020
+				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL 0xc6000020
+				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA 0xc6000020
 			>;
 		};
 
@@ -468,9 +467,8 @@
 
 		pinctrl_i2c0_mipi_lvds1: mipi_lvds1_i2c0_grp {
 			fsl,pins = <
-				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL	0xc6000020
-				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA	0xc6000020
-				SC_P_MIPI_DSI1_GPIO0_00_MIPI_DSI1_GPIO0_IO00	0x00000020
+				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL 0xc6000020
+				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA 0xc6000020
 			>;
 		};
 
@@ -1147,8 +1145,6 @@
 		reg = <0x3d>;
 		adi,dsi-lanes = <4>;
 		adi,dsi-channel = <1>;
-		interrupt-parent = <&gpio0>;
-		interrupts = <1 IRQ_TYPE_LEVEL_LOW>;
 		status = "okay";
 
 		port {
@@ -1235,8 +1231,6 @@
 		reg = <0x3d>;
 		adi,dsi-lanes = <4>;
 		adi,dsi-channel = <1>;
-		interrupt-parent = <&gpio0>;
-		interrupts = <1 IRQ_TYPE_LEVEL_LOW>;
 		status = "okay";
 
 		port {
-- 
1.7.9.5

